wiki/Tournaments/OWC/2021/en.md

```diff
@@ -0,0 +1,202 @@
+16. All players and staff must be treated with respect. Instructions of the referees and the Tournament Management are to be followed. Decisions labeled as final are not to be objected.
+17. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
+    - Breaking the chat rules may result in a silence. Silenced players cannot participate in multiplayer matches and must be exchanged for the duration of the punishment.
+18. Penalties for violating the tournament rules include, but are not limited to:
+    - Exclusion of specific players for one beatmap.
+    - Exclusion of specific players for an entire match.
+    - Declaring the match as forfeited, or as a win by default for the other team.
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future official tournaments, until appealed.
+19. Referees may allow, at their discretion, lower or higher tolerances for timers.
+20. The Tournament Management reserves the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Tournament registration
+
+1. Every user interested in joining their country's team signs up individually.
+   - Tournament Management will create a list of potential candidates for a country's team.
+   - Tournament Management will declare one candidate captain of their country's team.
+2. The declared captain will then form their team from the candidate list of their country.
+   - Captains are expected to choose team members with honesty and good will, with the aim of fielding the strongest team possible.
+   - Captains are allowed to exclude themselves from the team list.
+   - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms.
+   - Captains may choose, with the approval of the Tournament Management, at any time, to delegate their role to other prospective team members. Once given, only the new captain may assign the captaincy to another player.
+3. To ensure valid registrations, every prospective participant will be manually checked by the Tournament Management, in a manner similar to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening) that is offered to community tournaments.
+   - Every registered user will be assigned to their respective country's candidate list.
+   - To be successfully accepted on the list, players are required to be placed #5000 or higher on the global osu! performance ranking, and have not violated the [osu! community rules](/wiki/Rules) within the last 12 months.
```

Comment:
```suggestion
   - To be successfully accepted on the list, players are required to be placed #5000 or higher on the global osu! performance ranking, and have not violated the [osu! community rules](/wiki/Rules) within the last 12 months.
```

OWC 2020 used `better` rather than `higher` to describe rank limits, I think that it reads better that way also. Up to you though.

Reply:
Keeping it higher, that was exactly my thought process